charts/shoot-core/charts/monitoring/charts/blackbox-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: blackbox-exporter
+  namespace: kube-system
+  labels:
+    garden.sapcloud.io/role: system-component
+    component: blackbox-exporter
+    addonmanager.kubernetes.io/mode: Reconcile
+    origin: gardener
+spec:
+  revisionHistoryLimit: 0
+  replicas: 1
+  selector:
+    matchLabels:
+      component: blackbox-exporter
+  template:
+    metadata:
+      annotations:
```

Comment:
checksum for mounted secret is missing

Reply:
removed the secret so no need for a checksum.